### ISSUE TITLE
Spawn and despawn billboards

### ISSUE THREAD

**Issue by nixpulvis** [nixpulvis/galos#52]
Using `Visibility::Hidden` isn't enough to deal with the fact that they are children of each system. What was I thinking!

Using similar logic related to #43, we should be able to despawn and respawn billboards even while their parent system remains around.

**Comment by nixpulvis** [comment]
```rust
fn update(commands, camera: Query<&Camera>, systems: Query<(Entity, &System)>) {
    // forall systems, get camera distance then if:
    // distance > D remove label
    // else insert or reposition the label
}
```

**Comment by nixpulvis** [comment]
Fixed in ddbfd9131e7949be8da1c2b3dd659cd3c2df5304
